share/instruments/instruments.xml

```diff
@@ -2823,6 +2823,23 @@
                         <program value="60"/>
                   </Channel>
             </Instrument>
+            <Instrument id="horn">
```

Comment:
We already have a French Horn. It called Horn in F and already use the id "horn" and the musicxmlID brass.french-horn. If those are the same instrument "Horn in F" and "french horn" then let's pick one and not add another entry.